src/z3c/rml/pdfinclude.py

```diff
@@ -64,7 +67,7 @@ def do(cmd, cwd=None, captureOutput=True, ignoreErrors=False):
     if stderr is None:
         stderr = "See output above"
     if p.returncode != 0 and not ignoreErrors:
-        log.error(u'An error occurred while running command: %s' % cmd)
+        log.error('An error occurred while running command: %s' % cmd)
```

Comment:
Use format string.

Reply:
```suggestion
        log.error('An error occurred while running command: %s', cmd)
```

Isn't it still preferred to let the logger do the formatting lazily (because it might not be needed at all, and because the format can be customized)?

Reply:
There have been huge discussions about this on python-dev. Optimizations is the only real reason. But people have argued it is not worth it. I am indifferent. We can leave it as is.